src/components/Home.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+
+import getToken from '../lib/tokens';
+
+const Home = () => {
+  const [token, setToken] = useState(
+    JSON.parse(localStorage.getItem('token' || '')),
+  );
+
+  const history = useHistory();
+
+  useEffect(() => {
+    if (token) {
+      history.push('/list-view');
+    }
+    localStorage.setItem('token', JSON.stringify(token));
```

Comment:
local storage already adds the token as a string so there is no need to make it a string again. I suggest you remove the JSON.stringify() as it's adding unnecessary quotes.

Reply:
Hi @ciradu2204. We used the stringify method because local storage returns the value as null otherwise. Thank you!

Reply:
Hi @ciradu2204. We used the stringify method because local storage returns the value as null otherwise. Thank you!

Reply:
Good catch, we've removed it